opds_import.py

```diff
@@ -513,34 +462,23 @@ def combine(self, d1, d2):
     @classmethod
     def extract_data_from_feedparser(cls, feed, data_source):
         feedparser_parsed = feedparser.parse(feed)
-        values_meta = {}
-        values_circ = {}
+        values = {}
         status_messages = {}
         for entry in feedparser_parsed['entries']:
-            identifier, detail_meta, detail_circ, status_message = cls.data_detail_for_feedparser_entry(entry=entry, data_source=data_source)
+            identifier, detail, status_message = cls.data_detail_for_feedparser_entry(entry=entry, data_source=data_source)
 
             if identifier:
                 if status_message:
                     status_messages[identifier] = status_message
                 else:
-                    if detail_meta:
-                        values_meta[identifier] = detail_meta
-                    if detail_circ:
-                        values_circ[identifier] = detail_circ
+                    if detail:
+                        values[identifier] = detail
             else:
                 # That's bad. Can't make an item-specific error message, but write to 
                 # log that something very wrong happened.
                 logging.error("Tried to parse an element without a valid identifier.  feed=%s" % feed)
 
-        feed = feedparser_parsed['feed']
-        next_links = []
-        if feed and 'links' in feed:
-            next_links = [
-                link['href'] for link in feed['links'] 
-                if link['rel'] == 'next'
-            ]
-
-        return values_meta, values_circ, status_messages, next_links
+        return values, status_messages
```

Comment:
Next links are pulled out in a separate step, before import, so they're not needed here anymore.